playbooks/inventory/vagrant.py

```diff
@@ -40,7 +40,7 @@ def list_running_hosts():
 def get_host_details(host):
     cmd = "vagrant ssh-config {}".format(host)
     try:
-        saubprocess.check_call(cmd.split()).rstrip()
+        status = subprocess.check_output(cmd.split(), stderr=subprocess.STDOUT).rstrip()
```

Comment:
You can keep using check_call, but there's a typo in subprocess. Maybe only ```subprocess.CalledProcessError``` should be caught.